packages/sshnoports/scripts/install_sshnpd

```diff
@@ -35,6 +35,7 @@ usage() {
   echo "  -d, --device <address>      Device address (e.g. @alice_device)"
   echo "  -n, --name <device name>    Name of the device"
   echo "  -v, --version <version>     Version to install (default: latest)"
+  echo "      --args <args>           Additional arguments to sshnpd (\"-v -s -u\" by default)"
```

Comment:
You mentioned in the PR description, `supplying an empty string to args (--args "") will allow the exclusion of all non-mandatory fields.` ... I don't think that will be clear to the user from reading this usage help; maybe add that phrase to the message as well?

Reply:
How does a user get to know what `-v`, `-s` and `-u` are for?

Given recent discussions with @cconstab re #329 and #331 I feel like `-s` and `-u` should, by default, _not_ be set, so that the default security posture is the most defensive; but that needs to be accompanied by clear explanations of what those flags do during the install, so that the user gets the information and can then come back to it later on to adjust the sshnpd's arguments

Reply:
There's no good way for them to know... unless we duplicate the output of sshnpd into this script